File: solutions/problem9.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){
	// a, b, sqrt(a*a+b*b)
	// a+b+sqrt(a*a+b*b)=1000
	for(int a=1;a<=1000;a++){
		for(int b=a+1;b<=1000;b++){
			int c = sqrt(a*a+b*b);
			if(a+b+c==1000 && c*c==a*a+b*b){
				cout<<a<<" "<<b<<" "<<c<<endl;
				cout<<a*b*c<<endl;

			}
		}
	}
}

File: solutions/problem9.py
# a, b, sqrt(a*a+b*b)
# a+b+sqrt(a*a+b*b)=1000
for a in range(1,1000):
	for b in range(a+1,1000):
		c = int((a*a+b*b)**0.5)
		if(a+b+c==1000 and c*c==a*a+b*b):
			print(a,b,c)
			print(a*b*c)
